src/clr-angular/data/datagrid/datagrid-row.html

```diff
@@ -90,8 +90,8 @@
     </div>
     <div class="datagrid-row-scrollable" [ngClass]="{'is-replaced': replaced && expanded}">
       <div class="datagrid-scrolling-cells">
-        <ng-container #scrollableCells></ng-container>
         <ng-content select="clr-dg-cell"></ng-content>
+        <ng-container #scrollableCells></ng-container>
```

Comment:
Is this related to the issue you found with Anglar9 and the order of elements in a template?  Or something that has ben there since I added the scrollable and fixed columns?

Reply:
This is something related to Angular's quirky behavior. I learned that there is an actual difference between:
```
<ng-container></ng-container>
<ng-content></ng-content>
```
and
```
<ng-content></ng-content>
<ng-container></ng-container>
```